plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -23,6 +26,16 @@ const (
 	customAppType      = "SYSLOG"
 )
 
+var testLogger = func() telegraf.Logger {
+	if err := logger.SetupLogging(&logger.Config{Debug: true}); err != nil {
+		return nil
+	}
+	l := new(testutil.Logger)
+	sdkLog.Logger = logAdapter.NewLogger(l).WithGroup("tcg.sdk")
+
+	return l
+}()
+
```

Comment:
Please do not declare this globally! Anyway, this should not be necessary if you init the plugin in the tests, should it?

Reply:
Done, moved into plugin init